modules/dr-restoring-cluster-state-about.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/scenario-2-restoring-cluster-state.adoc
+
+[id="dr-scenario-2-restoring-cluster-state-about_{context}"]
+= About restoring cluster state
+
+You can use an etcd backup to restore your cluster to a previous state. This can be used to recover from the following situations:
+
+* The cluster has lost the majority of control plane hosts, leading to etcd quorum loss and the cluster going offline.
```

Comment:
clarify that this is quorum loss scenario 

`The cluster has lost the majority of control plane hosts (quorum loss)`